src/nft/components/details/AssetPriceDetails.tsx

```diff
@@ -342,6 +388,22 @@ export const AssetPriceDetails = ({ asset, collection }: AssetPriceDetailsProps)
       ) : (
         <NotForSale collectionName={collection.collectionName ?? 'this collection'} collectionUrl={asset.address} />
       )}
+      <OwnerInformationContainer>
```

Comment:
there's a lot more bottom padding under Seller on mobile compared to the other gaps